modules/storages/app/components/storages/admin/automatically_managed_project_folders_info_component.html.erb

```diff
@@ -0,0 +1,32 @@
+<%=
+  grid_layout('op-storage-view--row', tag: :div, align_items: :center) do |grid|
+    grid.with_area(:item, tag: :div, mr: 3) do
+      concat(render(Primer::Beta::Text.new(font_weight: :semibold, mr: 1, test_selector: 'storage-managed-project-folders-label')) { I18n.t('storages.file_storage_view.automatically_managed_folders') })
+      concat(automatically_managed_project_folders_status_label)
+    end
+
+    grid.with_area(:description, tag: :div, color: :subtle, test_selector: 'storage-automatically-managed-project-folders-description') do
+      render(Primer::Beta::Truncate.new(font_weight: :light)) { I18n.t('storages.page_titles.managed_project_folders.subtitle_short') }
+    end
+
+    if editable_storage?
+      grid.with_area(:"icon-button", tag: :div, color: :subtle) do
```

Comment:
The color should be `:muted` accoriding to Figma. This is especially important for the high-contrast mode. (Same is true for the other icons on the right side)